pkg/drivers/hyperkit/driver.go

```diff
@@ -240,6 +245,48 @@ func (d *Driver) Start() error {
 	return nil
 }
 
+func (d *Driver) checkForPidFile() error {
```

Comment:
checkForPidFile sounds like a function that simply asserts its existence. Since this does a bit more, please consider an alternative name and/or add a comment describing the goal of this function.